src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -31,16 +33,47 @@
     /** Length of a ansi defined color. */
     private static final int ANSI_PREFIX_LENGTH = 5;
 
-    public static String formatNotebook(String header, Notebook notebook) {
+    /**
+     * Method compiles the ArrayList items and appends the items to a String.
+     *
+     * @param notes ArrayList of notes to obtain note title/tags from
+     * @return noteString StringBuilder containing the notes ready to be printed
+     */
+    public static String formatNotes(String header, ArrayList<Note> notes) {
         String formattedString = "";
-        return formattedString;
+        return getNoteString(header, notes, formattedString);
```

Comment:
No need to have the formattedString if its just passing an empty string into the function.

Reply:
In that case, this function would actually be redundant, could just call the getNoteString directly